src/Benchmarks.ServerJob/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "description": "Class to transfer data to/from the benchmark server.",
```

Comment:
A separate library seems to more correctly express the relationship between components, so I think it should always be chosen over shared sources, unless there's a strong reason to do otherwise.